src/service/repo/signedMessageRepo.ts

```diff
@@ -12,6 +12,7 @@ type SendPayloadType = {
   message: string;
   channelType: number;
   messageType?: string;
+  attachments?: any;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `attachments` property is of type `any`. This could lead to potential issues as it doesn't provide any type safety. It would be better to define an interface for the attachments, even if it's a simple one.
```diff
-  attachments?: any;
+  attachments?: Attachment[];
```
Where `Attachment` is a new interface that you should create according to your needs.



<!-- This is an auto-generated comment by OSS CodeRabbit -->